svthalia/Reaxit
# Show external events on welcome screen

Closes #385.

### Summary
Shows external events on welcome screen.

### How to test
Steps to test the changes you made:
1. Go to welcome screen.
2. See that external evets are shown.
